src/playlist-loader.js

```diff
@@ -227,6 +228,44 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     }
   };
 
+  /**
+   * Returns the number of enabled playlists on the master playlist object
+   *
+   * @return {Number} number of eneabled playlists
+   */
+  loader.enabledPlaylists = function() {
```

Comment:
I'd err towards markings methods private unless you have a good use-case for adding them to our public API. We indicate private methods in video.js projects by appending an underscore to the method name. You could also convert this into a function by taking the master playlist as the first argument and then just skip exporting it.